model/src/funTest/kotlin/utils/PostgresProvenanceFileStorageFunTest.kt

```diff
@@ -60,16 +61,23 @@ class PostgresProvenanceFileStorageFunTest : WordSpec({
         }
 
         "return true when data for the given provenance has been added" {
-            storage.putData(VCS_PROVENANCE, InputStream.nullInputStream())
+            storage.putData(VCS_PROVENANCE, InputStream.nullInputStream(), 0L)
```

Comment:
Do you have a reference which shows that / why the size is required for postgres?